js/filters.js

```diff
@@ -6,6 +6,14 @@
   var housingType = mapFilters.querySelector('#housing-type');
   var onChange = null;
 
+  var filterHousingType = function (advert) {
+    return housingType.value === 'any' || housingType.value === advert.offer.type;
+  };
+
+  var check = function (advert) {
+    filterHousingType(advert);
```

Comment:
https://github.com/htmlacademy-javascript/987137-keksobooking-19/blob/490e45d0daff5eab267a9d6c16c901d2e9a6a179/js/page.js#L56
```suggestion
    return filterHousingType(advert);
```

Reply:
заметила, только когда отправила 🤦